5.0/fr/0x21-V12-Secure-Communication.md

```diff
@@ -0,0 +1,57 @@
+# V12 Secure Communication
+
+## Objectif de contrôle
+
+Ce chapitre comprend les exigences relatives aux mécanismes spécifiques qui doivent être mis en place pour protéger les données en transit, à la fois entre un client utilisateur final et un service backend, ainsi qu'entre les services internes et backend.
+
+Les concepts généraux promus par ce chapitre comprennent :
+
+* S’assurer que les communications ne sont pas interceptées par des parties non autorisées grâce à l’utilisation de certificats signés.
+* Configurer les mécanismes de chiffrement selon les dernières directives, y compris les algorithmes et chiffrements préférés.
+* Vérifier que les communications ne sont pas interceptées par une partie non autorisée à l'aide de certificats signés.
```

Comment:
There is some ambiguity there (which may be in the original text). Which one is it?


* Vérifier que les communications ne sont pas [interceptées par une partie non autorisée à l'aide de certificats signés].
* Vérifier que les communications ne sont pas [interceptées par une partie non autorisée] à l'aide de certificats signés.